components/cluster/command/reload.go

```diff
@@ -154,13 +168,57 @@ func buildReloadTask(
 			spec.ClusterPath(clusterName, "ssh", "id_rsa"),
 			spec.ClusterPath(clusterName, "ssh", "id_rsa.pub")).
 		ClusterSSH(metadata.Topology, metadata.User, gOpt.SSHTimeout).
-		Parallel(refreshConfigTasks...)
+		Parallel(refreshConfigTasks...).
```

Comment:
We can use `ParallelStep` to substitute the `Parallel` and make them more consistent.